articles/quickstart/native/ionic-react/01-login.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Login
+default: true
+description: This tutorial demonstrates how to add user login to an Ionic application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - native
+  - ionic
+  - react
+  - capacitor
+github:
+  path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD041 -->
+
+<%= include('../_includes/_getting_started', { library: 'Ionic' }) %>
+
+### Add a Platform
+
+If you have not already done so, use the CLI and Capacitor to add a supported mobile platform to your app. Use the following commands to add your desired platform(s) (e.g., `ios` or `android`):
+
+```bash
+# Add iOS
+npx cap add ios
+
+# Add Android
+npx cap add android
+```
+
+:::note
+For more information on adding Capacitor platforms to your app and the development workflow, check out the [getting started docs](https://capacitorjs.com/docs/v2/getting-started) as well as the starter docs for [iOS](https://capacitorjs.com/docs/v2/ios) and [Android](https://capacitorjs.com/docs/v2/android).
+:::
+
+<%= include('../../../_includes/_callback_url') %>
+
+The **Callback URL** to be used for your application includes your app's package ID which is found in the `config.xml` file for your app.
+
+Go to the [Application Settings](${manage_url}/#/applications/${account.clientId}/settings) section in your Auth0 dashboard and set your **Callback URL** in the **Allowed Callback URLs** box.
+
+You should set the **Allowed Callback URL** to
+
+```bash
+# replace YOUR_PACKAGE_ID with your app package ID
+YOUR_PACKAGE_ID://${account.namespace}/cordova/YOUR_PACKAGE_ID/callback
+```
+
+Replace `YOUR_PACKAGE_ID` with your application's package name.
+
+<%= include('../../../_includes/_logout_url') %>
+
+You should set the **Allowed Logout URLs** to
+
+```bash
+# replace YOUR_PACKAGE_ID with your app package ID
+YOUR_PACKAGE_ID://${account.namespace}/capacitor/YOUR_PACKAGE_ID/callback
```

Comment:
Should the logout url be `/callback` too?

Reply:
I had debated when writing the sample to use separate URLs for callback and logout, and in the end went with one. I don't have massively strong feelings on it, other than it makes configuration slightly simpler just having one. What are your thoughts?

Reply:
👍 1 url better than 2 urls